docs/porting-addons-to-v2.md

```diff
@@ -125,6 +128,14 @@ The steps:
    ember s
    ember test
    ```
+
+1. The lint scripts are expected to work the same way as before inside the test-app. However, there's one common issue you may encounter when running the linter if you use `eslint-plugin-n` or `eslint-plugin-node`: 
+   ```sh
+   error  "@embroider/test-setup" is not published  n/no-unpublished-require
+   error  "ember-cli" is not published  n/no-unpublished-require
+   ```
+   The [lint rule](https://github.com/eslint-community/eslint-plugin-n/blob/master/docs/rules/no-unpublished-require.md) tells that `"@embroider/test-setup"` and `"ember-cli"` are `devDependencies` being imported with `require()`. It's not a problem since the test-app is a private package. To solve the issue, make sure you use an up-to-date version of `eslint-plugin-n` (at least `15.4.0`). If you don't want to update the lint tools right now, you can also deactivate the rule.
```

Comment:
Oh i see! It's an issue with eslint-plugin-node. I gotchya.

Yeah, that's abandoned. Thanks for making these updates! I hope they save others' time in the future!